src/next-edge/index.ts

```diff
@@ -116,6 +116,11 @@ export interface CreateApiHandlerOptions {
    * If you need to forward additional headers you can use this setting to define them.
    */
   forwardAdditionalHeaders?: string[]
+
+  /*
+   * You can pass your next.config.js basePath here if your app lives on a subpath of the domain.
+   */
+  appBasePath?: string
```

Comment:
Would it be possible to instead load this from next itself directly? That way we can avoid another config parameter and it would just work with *magic*? :)